src/OpenLoco/src/Window.h

```diff
@@ -78,9 +78,23 @@ namespace OpenLoco::Ui
         constexpr uint8_t both = horizontal | vertical;
     }
 
+    enum class ScrollFlags : uint16_t
```

Comment:
could you spin off these 3 scroll things into a separate header. Just make it get included by Window.h for now. Might be better to do it as a separate PR.